package.json

```diff
@@ -3,20 +3,6 @@
   "version": "0.1.0",
   "repository": "https://github.com/danskernesdigitalebibliotek/dpl-design-system",
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
Goes without saying that you have tested this move of dependencies? :)

Reply:
Yep :smile:

`npm run dev` still work as expected and releases contain the same files as before.

Reply:
👍